modules/countdown.js

```diff
@@ -3,46 +3,78 @@ require('moment-countdown')
 
 let timeout = null
 
-const ARTICLE_50 = '2020-01-31T00:00:00+01:00'
+function isPast (date) {
+  const now = moment()
+  const expiryDate = moment(date)
+  return expiryDate.isBefore(now)
+}
+
+function nextDate (milestones) {
+  for (const milestone in milestones) {
+    if (!isPast(milestone.date)) return milestone
+  }
+  return { name: null, date: null }
+}
+
+function nextBrexitDate () {
+  const milestones = [
+    { name: 'Article 50', date: '2020-01-31T00:00:00+01:00' },
+    { name: 'The transition period', date: '2019-12-31T00:00:00+01:00' }
```

Comment:
Is there a formal name we can use here? Should we add the extension-request deadline as a milestone?

Reply:
Article 126 calls it the "transition or implementation period"; elsewhere in the WA "transition period" seems to be preferred.